info.puredata.Pd.yml

```diff
@@ -22,6 +22,25 @@ rename-desktop-file: org.puredata.pd-gui.desktop
 modules:
   - modules/tcl.json
   - modules/tk.json
+  - name: pd64
+    buildsystem: autotools
+    config-opts:
+      - --enable-jack
+      - --with-floatsize=64
+      - --program-transform-name='s/pd$/pd64/'
+    post-install:
+      - appstream-util modify "${FLATPAK_DEST}/share/metainfo/org.puredata.pd-gui.metainfo.xml" id "${FLATPAK_ID}"
```

Comment:
this will put the same app-id as the main one. definitely NOT what you want.

Reply:
it's actually just a single application with one `metainfo.xml` and a single app-id.

we are building two variants of the same program (and you can chose the variant from the GUI)

I am running the `appstream-util modify` and rename dance (until upstream has changed the app-id), so we don't end up with two conflicting metainfo files.

Reply:
> this will put the same app-id as the main one. definitely NOT what you want.

and just to stress: this will put the *same app-id* into the *same file* as the main one, producing identical results.
this *is* what we want.

(we could just ignore the `appstream-util` part altogether and rely on the next module overwriting the file anyhow.
but it feels cleaner that way (as you can disable the following module and still get a valid result)

Reply:
like on line 49?

I guess basicaly clobber the other.

Reply:
i'm not sure i understand (non-native speaker here).

yes, https://github.com/flathub/info.puredata.Pd/blob/ab5e86e3ac6a14ec652940e80f60496370ba989a/info.puredata.Pd.yml#L49 is what i had in mind when i said "the following module will overwrite the file anyhow.

dunne what you mean with "I guess basicaly clobber the other."

(nevertheless i strived for the the module https://github.com/flathub/info.puredata.Pd/blob/ab5e86e3ac6a14ec652940e80f60496370ba989a/info.puredata.Pd.yml#L25 to be identical to module https://github.com/flathub/info.puredata.Pd/blob/ab5e86e3ac6a14ec652940e80f60496370ba989a/info.puredata.Pd.yml#L44 except for the configure args.

Reply:
does this clarify (and resolve) the issue?